enterprise/cmd/coder/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+	_ "time/tzdata"
+
+	"github.com/coder/coder/cli"
+	"github.com/coder/coder/cli/cliui"
+)
+
+func main() {
+	rand.Seed(time.Now().UnixMicro())
+
+	cmd, err := cli.Root().ExecuteC()
```

Comment:
Do we plan on having a separate CLI package to maintain? How do we imagine this working?

Reply:
There will eventually be a separate enterprise `cli` package, because CLI commands for enterprise features will be in it.  It will import all the core commands from the AGPL `cli` package.